frontend/src/components/contact/contact_us.js

```diff
@@ -1,5 +1,65 @@
-import React from 'react';
+import React from "react";
+import { Button2 } from "../util/button/button";
+import "./contact_us.css";
+const ContactUs = () => {
+  return (
+    <div class="contact-section">
+      <div class="contact-parent">
+        <div class="contact-child child1">
+          <img src="./images/contact.jpg" alt="" className="contact-image" />
+        </div>
 
-const ContactUs = () => <h1 className='contact-us'>CONTACT US</h1>;
+        <div class="contact-child child2">
+          <h3 class="contact-header-text">Get in touch</h3>
+          <div class="inside-contact">
+            <div className="contact-input">
+              <input
+                id="txt_name"
+                type="text"
+                Required="required"
+                name="name"
+                placeholder="Your Name"
+              />
+              <i className="fas fa-user"></i>
+            </div>
+            <div className="contact-input">
+              <input
+                id="txt_email"
+                type="text"
+                Required="required"
+                name="email"
+                placeholder="Your Email"
+              />
+              <i className="far fa-envelope"></i>
+            </div>
+            <div className="contact-input">
+              <input
+                id="txt_subject"
+                type="text"
+                Required="required"
+                name="subject"
+                placeholder="Your Subject"
+              />
+              <i className="fas fa-pen-square"></i>
+            </div>
+            <div className="contact-input">
+              <textarea
+                id="txt_message"
+                rows="4"
+                cols="20"
+                name="message"
+                Required="required"
+                placeholder="Your Message"
+              ></textarea>
+              <i className="far fa-comment-alt"></i>
+            </div>
+
+            <Button2 label="Submit" type="submit" />
```

Comment:
Can we please have in the center of the right side form

Reply:
Yes I have updated the requested changes in the recent commit.